core/templates/dev/head/components/profileLink.js

```diff
@@ -0,0 +1,21 @@
+oppia.directive('profileLinkText', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      linkedName: '@'
+    },
+    templateUrl: 'components/profileLinkText',
+  };
+}]);
+
+oppia.directive('profileLinkImage', [function() { 
+  return {
+    restrict: 'E',
+    transclude: true,
+    scope: {
+      linkedName: '@',
+      linkedImage: '@'
```

Comment:
Yeah I knew transclude was maybe overkill-- I ended up using it because of all the logic in base.html around the drop down, and I didn't want to mess anything up. If I take the profile jmage link usage out of the navbar though (which it sounds like I will from your other comments) I think those concerns are resolved, and I can construct an img passing in the appropriate class attributes as an attribute.